File: bootcamp_c_cpp_dsa-main/bootcamp_c_cpp_dsa-main/C+DS/codes/Recursion/p27.c
# include <stdio.h>
# include <stdbool.h>
bool demo (int arr[],int size) {
	int count=0 ;
	for (int i=0 ; i<size ; i++) {
		if (arr[i]%2==0)
			count++ ;
		if (count>=2)
			return true ;
	}
	return false ;
}
void main () {
	int size ;
	printf("Enter size of an array\n") ;
	scanf("%d",&size) ;

	int arr[size] ;
	printf("Enter elements in an array\n") ;
	for (int i=0 ; i<size ; i++) {
		scanf("%d",&arr[i]) ;
	}

	bool ret=demo(arr,size) ;
	if (ret==true) 
		printf("Found\n") ;
	else 
		printf("Not found\n") ;
}

File: java-main/java-main/DSA/Array/p27.java
class Demo{
        public static void main(String[]args){
                int arr[][]=new int [][]{{1,2,3,4},{5,6,7,8},{9,10,11,12}};
                int i=0;
                int j=arr.length-1;
                while(i<arr.length && j>=0){
                        System.out.println(arr[i][i]);
                        i++;
                        j--;
                }
        }
}


